apps/core/migrations/0006_footercontent_footeritem.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.0.7 on 2022-10-23 06:39
+
+from django.db import migrations, models
+import django.db.models.deletion
+import modelcluster.fields
+import wagtail.fields
+
+
+class Migration(migrations.Migration):
```

Comment:
Could you please add a separate migration that creates the current content as an instance of this snippet? If not I can add it later when I merge it

Reply:
Hey @laymonage . I could not exactly understand what you meant here so I have left this task 😅 . Could you please elaborate on what should I do here? Or if you want you can pick this up.

Reply:
We can write our own migration and use `migrations.RunPython` to create an initial instance of the snippet based on the content that we have now. For reference: https://github.com/wagtail/guide/pull/175/commits/7bb24aabd440284532e26a064574bf6144ed4d31#diff-18e703351254cbef277b981e328c2a4941262dd85442cdd93c980255b76d67ba

But no worries, I'm happy with the PR overall. I'll write the migration along with some minor fixes to the styling of links in the footer text 😄